Framework/Properties/Mantid.properties.template

```diff
@@ -54,6 +54,8 @@ CheckMantidVersion.DownloadURL = http://download.mantidproject.org
 
 # Whether to report usage statistics back to central server
 usagereports.enabled = @ENABLE_USAGE_REPORTS@
+error_root_url = http://errorreports.mantidproject.org/api/error
+usage_root_url = http://reports.mantidproject.org
```

Comment:
Can we follow the standard property config style for names, please?

I think this should be:

```
usagereports.rooturl =
```

and

```
errorreports.rooturl = 
```

Also I think the root errors should be "http://errorreports.mantidproject.org" and then the `api/error` appended internally so that both are consistent. If I'm testing then I just want to be able to change that to `localhost` and have it work.